linux/start/jupyter/adoc/gs_sles_jupyter-jupyterlab_container.adoc

```diff
@@ -547,19 +547,18 @@ WORKDIR /home/jupyter
 COPY --chown=1000:1000 requirements.txt .
 
 # install JuypterLab
-RUN set -eux pipefail; \
+RUN set -euxo pipefail; \
     pip install --upgrade pip; \
     pip install -r requirements.txt; \
     pip cache purge;
 
 # expose JupyterLab web port
 EXPOSE 8888/tcp 
 
-# change to notebooks directory
-RUN cd $NOTEBOOKS
+VOLUME $NOTEBOOKS
```

Comment:
The guide uses a "launch script" to handle volume mapping when the container is run.
Adding the VOLUME command to the Container file is an unnecessary complication.

The original lines 558 and 559 are irrelevant and can be removed.